### PR TITLE
Add full testing suite for BanishManager

### PR DESCRIPTION
**Pull request by gausie**
I want to refactor BanishManager, so here are some tests for it to make sure nothing breaks.

### CODE REVIEW

File: test/net/sourceforge/kolmafia/session/BanishManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+    assertTrue(BanishManager.isBanished("paper towelgeist"));
+
+    assertFalse(BanishManager.isBanished("zmobie"));
+  }
+
+  @Test
+  void saveBanishedMonsters() {
+    assertThat("banishedMonsters", isSetTo(""));
+
+    // This will be removed because its run out.
```
Comment (midgleyc) [inline test/net/sourceforge/kolmafia/session/BanishManagerTest.java:71]: `it's`, as in `it has`. Or just in long.

**Review by midgleyc — APPROVED** [review]
wow, good job with these

**Comment by midgleyc** [comment]
You might want to add a test to check the Banish Manager doesn't error horribly in case of an unparseable preference, but I don't think it's necessary to block refactoring work (as I don't think it needs to work the same as it did, necessarily -- just "don't crash").

File: test/net/sourceforge/kolmafia/session/BanishManagerTest.java (hunk 1 above)
Comment (gausie) [reply test/net/sourceforge/kolmafia/session/BanishManagerTest.java:71]: Oh yeah, I'll fix this next round

**Pull request merged**